core/controllers/base_test.py

```diff
@@ -333,6 +330,18 @@ def _mock_logging_function(msg, *args):
 
         self.assertEqual(observed_log_messages, ['Frontend error: errors'])
 
+    def test_redirect_when_user_is_disabled(self):
+        # The old demo server redirects to the new demo server.
```

Comment:
How does this comment relate to what follows?

Reply:
It doesn't, removed.